agent/src/main/java/org/openjdk/jmc/agent/Agent.java

```diff
@@ -102,7 +101,8 @@ public static void agentmain(String agentArguments, Instrumentation instrumentat
 	 */
 	public static void initializeAgent(InputStream configuration, Instrumentation instrumentation)
 			throws XMLStreamException {
-		TransformRegistry registry = DefaultTransformRegistry.from(configuration);
+		TransformRegistry registry =
+				configuration != null ? DefaultTransformRegistry.from(configuration) : DefaultTransformRegistry.empty();
```

Comment:
Nit: should document that null as argument for the configuration stream means using the empty configuration.

Reply:
Thank you. Fixed in b7e05c347f9f634f6d52e9639c619d66c74e9576